app/src/main/java/com/jnj/vaccinetracker/visitsoverview/model/VisitsListViewModel.kt

```diff
@@ -169,6 +180,13 @@ class VisitsListViewModel @Inject constructor(
         }
     }
 
+//    fun isInternetAvailable(context: Context): Boolean {
```

Comment:
Yeah lets get rid of these comments and un used imports and get this merged. Otherwise well done.